File: CourseCodesAPI/CourseCodesAPI/Models/SolutionRunRequest.cs
using System;

namespace CourseCodesAPI.Models
{
	public class SolutionRunRequest
	{
		public Guid SolutionId { get; set; }
		public string SourceCode { get; set; }
		public Guid StudentId { get; set; }
		public Guid ProblemId { get; set; }
	}
}

File: coursecodesapp/src/api/models/SolutionRunRequest.js
export class SolutionRunRequest {
	constructor({ solutionId, sourceCode, studentId, courseProblemId }) {
		this.solutionId = solutionId;
		this.sourceCode = sourceCode;
		this.studentId = studentId;
		this.courseProblemId = courseProblemId;
	}
}
